sources/lib/ParameterJuicer.php

```diff
@@ -281,7 +287,7 @@ private function setDefaultValues(array $values): array
     {
         foreach ($this->default_values as $field => $default_value) {
             if (!isset($values[$field]) && !array_key_exists($field, $values)) {
-                $values[$field] = $default_value;
+                $values[$field] = $default_value();
```

Comment:
Does this have the same behaviour as using `call_user_func`? I have searched through the documentation but could not find if this uses `call_user_func` internally.